udata/assets.py

```diff
@@ -73,10 +73,15 @@ def from_manifest(app, filename, raw=False, **kwargs):
     path = _manifests[app][filename]
 
     if not raw and cfg.get('CDN_DOMAIN') and not cfg.get('CDN_DEBUG'):
-        prefix = 'https://' if cfg.get('CDN_HTTPS') else '//'
```

Comment:
Has the logic with `//` been removed from this PR? I think I remember we needed that in some case (like on embeds).

Reply:
Yes, because it wasn't working with some protocols like `file://` (local testing, standalone HTML page).
Tested with embeds (this is while testing it I noticed this issue)

Reply:
Don't you get mixed content warning/error in some cases?

Reply:
It shouldn't because `request.scheme` is in sync with the incoming request scheme.
I tested locally with:
- http without CDN
- https without CDN
- http with http CDN
- http with https CDN
- https with HTTPS CDN
Tested in siteassets and oembed